app/models/application_form.rb

```diff
@@ -53,6 +53,7 @@ class ApplicationForm < ApplicationRecord
   has_many :documents, as: :documentable
   has_many :timeline_events
   has_one :assessment
+  has_many :notes
```

Comment:
Should the notes be associated with the assessment rather than the application_form? I'm not sure it matters really at the moment. Just wondering.

Reply:
Yes, initially I was going to put it on the assessment, but then I started to think about how it was going to be used, and I realised notes could be added at any point in the process, since they're generic, and not necessarily tied to a particular assessment (especially if we start having multiple assessments for reviewers, etc). I figured since notes could be used for anything, I didn't want to tie them to an assessment.